docs/api/crash-reporter.md

```diff
@@ -43,45 +46,68 @@ The `crashReporter` module has the following methods:
-reports temporarily. You can test this out by calling `process.crash()` to crash the child process.
-
-**Note:** If you need send additional/updated `extra` parameters after your
-first call `start` you can call `addExtraParameter` on macOS or call `start`
-again with the new/updated `extra` parameters on Linux and Windows.
-
-**Note:** On macOS and windows, Electron uses a new `crashpad` client for crash collection and reporting.
-If you want to enable crash reporting, initializing `crashpad` from the main process using `crashReporter.start` is required
-regardless of which process you want to collect crashes from. Once initialized this way, the crashpad handler collects
-crashes from all processes. You still have to call `crashReporter.start` from the renderer or child process, otherwise crashes from
-them will get reported without `companyName`, `productName` or any of the `extra` information.
+  * `extra` Record<String, String> (optional) - Extra string key/value
+    annotations that will be sent along with crash reports that are generated
+    in the main process. Only string values are supported. Crashes generated in
+    child processes will not contain these extra
+    parameters to crash reports generated from child processes, call
+    [`addExtraParameter`](#crashreporteraddextraparameterkey-value) from the
+    child process.
+  * `globalExtra` Record<String, String> (optional) - Extra string key/value
+    annotations that will be sent along with any crash reports generated in any
+    process. These annotations cannot be changed once the crash reporter has
+    been started. If a key is present in both the global extra parameters and
+    the process-specific extra parameters, then the global one will take
+    precedence. By default, `productName` and the app version are included, as
+    well as the Electron version.
+
+This method must be called before using any other `crashReporter` APIs. Once
+initialized this way, the crashpad handler collects crashes from all
+subsequently created processes. The crash reporter cannot be disabled once
+started.
+
+This method should be called as early as possible in app startup, preferably
+before `app.on('ready')`. If the crash reporter is not initialized at the time
+a renderer process is created, then that renderer process will not be monitored
+by the crash reporter.
+
+**Note:** You can test out the crash reporter by generating a crash using
+`process.crash()`.
+
+**Note:** If you need to send additional/updated `extra` parameters after your
+first call `start` you can call `addExtraParameter`.
+
+**Note:** Parameters passed in `extra`, `globalExtra` or set with
+`addExtraParameter` have limits on the length of the keys and values. Key names
+must be at most 64 bytes long, and values must be no longer than 128 bytes.
+Keys with names longer than the maximum will be silently ignored. Key values
```

Comment:
What is the behavior when two keys longer than 64 bytes truncate to the same key, does the collision cause other issues or will one of them just win out.

Reply:
keys aren't required to be unique, so you'll get 2 in the report.